homeassistant/components/netgear/sensor.py

```diff
@@ -374,4 +377,12 @@ def async_update_device(self) -> None:
         """Update the Netgear device."""
         if self.coordinator.data is not None:
```

Comment:
We can invert the check here and return early if true. Then we can outdent below.

Reply:
Thanks for the tip, this makes it a lot more structured.